Fix duplicate main function call

Moonkicker64 opened the pull request:
The main function is called twice.
This commit removes the redundant call to ensure the program executes correctly.
Let me know if you have any suggestions!

matiasdaloia commented:
Hi @Moonkicker64! Thanks for spotting the issue

However, a couple comments on the PR:
- Keep in mind that it includes changes from a demo branch. You should instead open a new branch from main and create the PR to main
- There are a lot of modifications because of linting/formatting. Please try not to commit these changes because it's really difficult to spot your actual fix. Also keep in mind we use single quotes for strings

Thanks and regards,

Matias

Moonkicker64 commented:
Thanks for reply.
I opened a pull request to the wrong repository.
The code I fixed only exists in the demo branch.
I have changed the base repository from main to demo-fosdem-2025.
Please review it.
